demo/paramLoader.js

```diff
@@ -54,6 +54,14 @@ export const s3 = bucket && region && name && accessKeyId && secretAccessKey &&
         sessionToken: sessionToken,
     });
 
+export const lambda = new AWS.Lambda({
+  region: region,
+  accessKeyId: accessKeyId,
+  secretAccessKey: secretAccessKey,
+  sessionToken: sessionToken,
+  apiVersion: '2015-03-31'
+});
+
```

Comment:
Yuck. Let's be getting rid of global variables, not adding more. Make a function that returns a new Lambda object.